cli/js/doc.ts

```diff
@@ -0,0 +1,297 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+import "./ts_global.d.ts";
+import "./globals.ts";
+
+import * as util from "./util.ts";
+
+const { assert } = util;
```

Comment:
Is it possible to remove these imports? That would make the code more portable to non-deno environments.